### PR TITLE
update more apps, add Thorium

### PR DESCRIPTION
**Pull request by Drsheppard01**

### CODE REVIEW

**Review by blueOkiris — CHANGES_REQUESTED** [review]
Other than the continuous thing for appimagetool, the rest looks good.

I will note a couple of these like LibreOffice may get new downloads before the version is increased to match, but that shouldn't cause any problems with upgrades. Just some people would potentially have a newer version than they thought they downloaded until the package list is corrected, so that's fine.

File: pkgs.json [hunk 1]
```diff
@@ -21,19 +21,19 @@
...
+        "url": "https://github.com/AppImageCommunity/AppImageUpdate/releases/download/2.0.0-alpha-1-20230304/AppImageUpdate-x86_64.AppImage"
     }, {
         "name": "appimaged",
         "version": "765.1",
         "description": "appimaged is an optional daemon that watches locations like ~/bin and ~/Downloads for AppImages and if it detects some, registers them with the system, so that they show up in the menu, have their icons show up, MIME types associated, etc.",
         "url": "https://github.com/blueOkiris/aip-man-pkgs/releases/download/appimaged-765/appimaged-765-x86_64.AppImage"
     }, {
         "name": "appimagetool",
-        "version": "13",
+        "version": "Continuous",
```
[the remark this answers is not in the document]
Comment (Drsheppard01) [reply pkgs.json:34]: I think that entering an internal number and occasionally updating the assembly and the number assigned would be more convenient.

[the remark this answers is not in the document]
Comment (blueOkiris) [reply pkgs.json:34]: They just released a new version (I'm watching it, apparently), so I uploaded that [here](https://github.com/blueOkiris/aip-man-pkgs/releases/download/appimagetool-2023.3.10.1)

You can use version "2023.3.10.1" and link here "https://github.com/blueOkiris/aip-man-pkgs/releases/download/appimagetool-2023.3.10.1/appimagetool-x86_64.AppImage"

Might be good to keep version 13 as "appimagetool-stable" or something as well though.

**Comment by blueOkiris** [comment]
I haven't seen any progress made, so I went ahead and updated the apps that were changed in this PR, and I'm closing this.

Here's the [monthly updates PR](https://github.com/blueOkiris/aip-man-pkg-list/pull/4)

If you have more changes feel free to open a new PR

**Comment by Drsheppard01** [comment]
Thanks, I've been a little busy, what 'bout the apps with the continous tag, I think it's better to release them with the tag which has format "continous-mm-dd-yyyy". This will help users to have the latest build with the latest commits, and aip-man will be able to parse the information and download the desired build

**Comment by blueOkiris** [comment]
That works for me. Aip-man just needs something to reference when the build disappears, so simply marking continuous and date makes sense.

I'll add a PR tomorrow to rename the relevant apps

**Pull request closed without merging**